### PR TITLE
optionally dump node output on fail of a node integration test

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

File: scripts/node_integration_tests/conftest.py [hunk 1]
```diff
@@ -36,17 +38,48 @@ def enabled(self):
         return REUSE_KEYS
 
 
+class DumpOutput:
+    @staticmethod
+    def on_crash():
```
Comment (kmazurek) [inline scripts/node_integration_tests/conftest.py]: This method name sounds like a handler, perhaps add a type hint here?

File: scripts/node_integration_tests/conftest.py [hunk 2]
```diff
@@ -36,17 +38,48 @@ def enabled(self):
...
+        return DUMP_OUTPUT_ON_CRASH
+
+    @staticmethod
+    def enable_on_crash():
+        global DUMP_OUTPUT_ON_CRASH
+        DUMP_OUTPUT_ON_CRASH = True
+
+    @staticmethod
+    def on_fail():
```
Comment (kmazurek) [inline scripts/node_integration_tests/conftest.py]: Same as above.

**Review by kmazurek — APPROVED** [review]
lgtm

File: scripts/node_integration_tests/conftest.py (hunk 1 above)
Comment (shadeofblue) [reply scripts/node_integration_tests/conftest.py]: right, I'll rather just change the name to something else, thanks :)

**Pull request merged**